mne/decoding/transformer.py

```diff
@@ -245,6 +245,97 @@ def inverse_transform(self, X, y=None):
         return X.reshape(-1, self.n_channels, self.n_times)
 
 
+class Vectorizer(TransformerMixin):
+    """Transforms n-dimensional array into 2D array of n_sample by n_features.
+
+    This class reshapes an n-dimensional array into an n_sample * n_features
+    array, usable by the estimators and transformers of scikit-learn.
+
+    Examples
+    --------
+    clf = make_pipeline(SpatialFilter(), XdawnTransformer(), Vectorizer(),
+                        LogisticRegression())
+
+    Attributes
+    ----------
+    ``shape_`` : array
+         Stores the original dimension of data.
```

Comment:
I agree with @mmagnuski. Also change `shape_` to `features_shape_` to be more explicit.